inductiva/tasks/task.py

```diff
@@ -227,10 +228,10 @@ def download_outputs(
         # the size of the file, etc.)
         response = api_response.response
 
+        output_dir = files.resolve_path(output_dir)
         if output_dir is None:
             output_dir = files.resolve_path(inductiva.output_dir).joinpath(
                 self.id)
-
         if output_dir.exists():
```

Comment:
Are we informing the user that we are deleting this directory?

Reply:
I think it is better not to delete it, actually, I am going to correct it.